### PR TITLE
Creating Recipe Page

### PR DESCRIPTION
**Pull request by jiajdo**

### CODE REVIEW

**Comment by eatnumber1** [comment]
Can you adjust the pull request's title to reflect the goal you're trying to achieve?

File: recipe/index.html
Comment (eatnumber1) [inline recipe/index.html:1]: Looks like you have a typo in this commit? (the trailing '>'). Can you fix?

Comment (jiajdo) [reply recipe/index.html:1]: Edited first 2 commit messages for typos and made each commit purpose more clear. Also edited pull request title.

**Review by eatnumber1 — CHANGES_REQUESTED** [review]
It's optional, as some projects require it and some don't, but it's considered good practice to keep the first line of your commit message under 50 characters.

You can see why in the way github renders the commit message for ef7d57dd2bec052afb29b0adc458cbc9e3fae5aa. Apparently github is more lax than 50 characters, but it still breaks the line at 110 characters.

So I'd recommend shortening the summary line of your message, and adding a short paragraph beneath if more detail is warranted.

File: recipe/index.html [hunk 1]
```diff
@@ -50,7 +50,7 @@ <h3>Second Step: Assemble</h3>
         <div>
         <h3>Other Recipes to Try</h3> 
         <p><b><a href = "https://www.culinaryhill.com/classic-tuna-salad-sandwich/">Tuna Salad</a></b></p>
-        <p><a href = "https://www.spendwithpennies.com/classic-chicken-salad-recipe/">Chicken Salad</a></p>
+        <p><b><a href = "https://www.spendwithpennies.com/classic-chicken-salad-recipe/">Chicken Salad</a></b></p>
```
Comment (eatnumber1) [inline recipe/index.html:53]: This doesn't do what the commit message says... this bolds the recipe, it doesn't add it.

**Review by eatnumber1 — CHANGES_REQUESTED** [review]

File: recipe/index.html (hunk 1 above)
Comment (jiajdo) [reply recipe/index.html:53]: Reworded commit message

**Comment by jiajdo** [comment]
Shortened summary line in commit [a8da806](https://github.com/jiajdo/jiatechtonicaexercises/pull/1/commits/a8da80683f016f32cd49ae5463eb83a8d4591936) added more information about the commit in newline paragraph.

**Review by eatnumber1 — APPROVED** [review]

**Pull request merged**